docs/admin/networking/troubleshooting.md

```diff
@@ -95,9 +95,8 @@ the NAT configuration, or deploy an internal STUN server.
 
 If a network interface on the side of either the client or agent has an MTU
 smaller than 1378, any direct connections form may have degraded quality or
-performance, as IP packets are fragmented. `coder ping` will indicate if this is
-the case by inspecting network interfaces on both the client and the workspace
-agent.
+might hang entirely. `coder ping` will indicate if this is the case by inspecting
+network interfaces on both the client and the workspace agent.
 
 If another interface cannot be used, and the MTU cannot be changed, you may need
```

Comment:
```suggestion
If another interface cannot be used, and the MTU cannot be changed, you should
disable direct connections and relay all traffic via DERP instead, which
will not be affected by the low MTU.
To disable direct connections, set the
[`--block-direct-connections`](../../reference/cli/server.md#--block-direct-connections)
flag or `CODER_BLOCK_DIRECT` environment variable on the Coder server.
```

Reply:
these suggestions are starting to feel a little chaotic. if you're cool with this change, let me know and I'll just push a commit to replace LL101-103

Reply:
yeah, sounds good, push away.